Symfony/CS/Tests/Fixer/Contrib/ProtectedToPrivateFixerTest.php

```diff
@@ -48,47 +48,47 @@ private static function f6(){}
         $attributesAndMethodsFixed = str_replace('protected', 'private', $attributesAndMethodsOriginal);
 
         return array(
-            'final-extends' => array(
```

Comment:
@SpacePossum I find this keys very useful when the test fails because it helps identifing the case. Why removing them? To be more similar to other fixer tests?
Btw all other changes are good :+1: